README.md

```diff
@@ -32,6 +32,15 @@ and makes them available to recruiters in the form of subscription.
 1. Navigate to backend/app folder
 1. `flask db-manage remove-data`
 
+### How to run automatic data scraping from gh
+
+1. Navigate to `backend/app folder`
+2. Update file named `GH_seed_repos.txt` - list of repos to scrape (optional)
```

Comment:
Change this hardcoded file so that its path is stored in the environment variable for easier dpeloyment